src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+
+  const arraySource = sourceString.split(';');
+
+  for (const key of arraySource) {
+    const [property, value] = key.split(':')
+      .map(str => str.trim());
```

Comment:
```suggestion
    const [property, value] = key
      .split(':')
      .map(str => str.trim());
```
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/8c3a5110-70e0-41f5-8226-f52a45d978e9)